changelog/upgrade-notes.md

```diff
@@ -116,6 +116,10 @@
 with the `imgSize` option and not with `size`. `size` is supposed to be used for the
 size of a sub-rectangle in an image sprite.
 
+### Support for non-square tiles
+
+When using ´ol.tilegrid.TileGrid#getTileSize` on the tile grid of an `ol.source.BingMaps`, the return value will now be an `ol.Size` array instead of a number.
```

Comment:
Good point. I updated the upgrade notes to explain the behaviour of `#getTileSize()`.